input/examples/practitioner-darleen-mackay.xml

```diff
@@ -4,14 +4,24 @@
         <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitioner"/>
     </meta>
     <identifier>
-        <system value="http://www.acme.org/practitioners"/>
-        <value value="254567"/>
+        <type>
+            <coding>
+                <system value="http://terminology.hl7.org/CodeSystem/v2-0203"/>
+                <code value="EI"/>
+            </coding>
+            <text value="Employee number"/>
+        </type>
+        <system value="http://nephrologyclinic.org.example.com/employee-id"/>
+        <value value="N765009-mackay-darleen"/>
+        <assigner>
+            <display value="Nephrology Clinic"/>
```

Comment:
reads like type of organisation not name. Make it Docker River Nephrology Clinic.